docs/pages/build/building-from-github.mdx

```diff
@@ -8,9 +8,11 @@ import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 
 This guide explains how to trigger builds directly from your GitHub repository using the Expo GitHub App.
 
+> **warning** This feature is in early access and may change significantly. Its performance might
+> not be as polished or reliable as our established features during this early access period. We appreciate your understanding and welcome any feedback to help us improve!
```

Comment:
```suggestion
> **warning** This feature is in early access and may change significantly. Its performance might not be as polished or reliable as our established features during this early access period. We appreciate your understanding and welcome any feedback to help us improve!
```

I'd suggest to not split lines here as the formatting is usually handled by Prettier 🙂